boa_engine/src/tests.rs

```diff
@@ -2993,3 +2993,138 @@ fn unary_operations_on_this() {
         assert!(string.contains(pos));
     }
 }
+
+#[test]
+fn try_break_finally_edge_cases() {
```

Comment:
Some of the scripts have spaces before the end of the line. Is that intentional?

Reply:
Nope! Definitely not. Thanks for catching that!